src/actinia_core/resources/common/config.py

```diff
@@ -478,5 +478,13 @@ def read(self, path=DEFAULT_CONFIG_PATH):
                     self.S3_AWS_RESOURCE_BUCKET = config.get(
                         "AWS_S3", "S3_AWS_RESOURCE_BUCKET")
 
+        # Overwrite values with environment variables if exist:
+        if os.environ.get('REDIS_SERVER_URL'):
```

Comment:
Maybe a warning `if config.has_option("REDIS", "REDIS_SERVER_URL")` is also given would be nice that says that the env values are taken. But that is not so important to me otherwise, you may merge. Thanks for the change :)